### PR TITLE
 Make S3 dependency optional

### PR DESCRIPTION
**Pull request by fviale**

### CODE REVIEW

**Review by MaelAudren — APPROVED** [review]

File: build.gradle [hunk 1]
```diff
@@ -284,9 +284,11 @@ project('programming-extensions:programming-extension-vfsprovider') {
                 project(':programming-extensions:programming-extension-dataspaces-common')
         )
 
-        compile('activeeon:vfs-s3:2.4.8') {
-            exclude group: 'com.fasterxml.jackson.core'
-            exclude group: 'com.fasterxml.jackson.dataformat'
+        if (rootProject.hasProperty('s3')) {
```
Comment (nebilbenmabrouk) [inline build.gradle:289]: Would you add please a commented example about where and how to define the property 's3'.

Comment (fviale) [reply build.gradle:289]: @nebilbenmabrouk done

**Review by mcompastie — APPROVED** [review]
This perfectly matches the modifications I have made on the prototype for RPi. Let's see how its works when integrated in PA.

**Review by nebilbenmabrouk — APPROVED** [review]

**Comment by fviale** [comment]
Retest this please

**Pull request merged**